model/Dataset/Properties/sensors.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# sensors
+
+## Summary
+
+The field describes what sensors where used for collecting the data (if any sensors were used) 
+
+## Description
+
+Each sensor and their callibration value in form Sensor:calibration is noted. If multiple sensors are used, each sensor and thier callibration value is noted ina new line
```

Comment:
`sensors` has arbitrary cardinality when used in the [`Dataset` class](https://github.com/spdx/spdx-3-model/blob/dataset-profile/model/Dataset/Classes/Dataset.md), so we should not discuss multiple values in this property.